client/src/components/examples/RequestGroupFormContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import { bindActionCreators, Dispatch } from "redux"
+import { gql, useQuery } from "@apollo/client";
+import React, { FunctionComponent, useState } from "react";
+import { connect } from "react-redux";
+
+import { loadRequestGroups } from '../../data/actions'
+import RequestGroup from "../../data/types/requestGroup"
+import RequestGroupForm from "../organisms/RequestGroupForm";
+import { RootState } from '../../data/reducers'
+
+
+interface DispatchProps {
+  loadRequestGroups: typeof loadRequestGroups,
+}
+
+type Props = DispatchProps
+
+
+const RequestGroupFormContainer: FunctionComponent<Props> = (props: Props) => {
+  const [show, setShow] = useState(true);
+
+  const query = gql`
+  {
+    requestGroups {
+      _id
+      name
+    }
+  }`
+
+  useQuery(query, {
+    onCompleted: (data: { requestGroups: Array<RequestGroup> }) => {
+      // Clone state.data because sort occurs in-place.
+      console.log("All request groups:")
```

Comment:
Remove console log

Reply:
LOL this is the example usage container ;)) it's for reference

Reply:
LOL